client/containers/DashboardOverview/CollectionOverview/DashboardCollectionOverview.tsx

```diff
@@ -180,6 +200,13 @@ const DashboardCollectionOverview = (props: Props) => {
 	const renderSecondaryContent = () => {
 		return (
 			<>
+				<OverviewSection title="recently viewed">
```

Comment:
Let's remove the Recently Viewed section from the Collection overviews — since it's not scoped to the Collection, it may be a little misleading.